src/test/java/utils/seleniumWrapper.java

```diff
@@ -0,0 +1,35 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public class seleniumWrapper {
+
+    WebDriver driver = new EdgeDriver();
+
+       public boolean clickElement(By locator){
+        driver.findElement(locator).click();
+        return true;
+
+    }
+
+    //function 2 - isDisplayed
+    public boolean displayElement(By locator){
+        driver.findElement(locator).isDisplayed();
+        return true;
+    }
+
+    //function 3 - quit browser
+    public boolean quitElement(By locator){
```

Comment:
By locator is not being used inside the functions, so it should not be passed as a parameter here